core/computed/entity-classification.js

```diff
@@ -20,11 +20,13 @@ class EntityClassification {
    */
   static makeUpAnEntity(entityCache, url) {
     if (!UrlUtils.isValid(url)) return;
-    // We can make up an entity only for those URLs with a valid domain attached.
-    // So we further restrict from allowed URLs to (http/https).
-    if (!Util.createOrReturnURL(url).protocol.startsWith('http')) return;
 
-    const rootDomain = Util.getRootDomain(url);
+    const parsedUrl = Util.createOrReturnURL(url);
+    const isChromeExtension = parsedUrl.protocol.startsWith('chrome-extension');
+    // Make up an entity only for valid http/https URLs and Chrome extensions.
+    if (!isChromeExtension && !parsedUrl.protocol.startsWith('http')) return;
+
+    const rootDomain = isChromeExtension ? 'Chrome Extensions' : Util.getRootDomain(url);
```

Comment:
We don't need a new gatherer to get the extension's name. This computed artifact has the devtools log, so we just need to grab the `Runtime.executionContextCreated` that matches this URL.

Reply:
That's awesome. Do you think we should go ahead and classify each entity into its own group though?

Reply:
Each into their own group seems simpler - it'll be obvious which URL comes from what extension (without us needing to use a text detail that combines the URL + the ext name), and we won't need to translate any strings.

Reply:
Updated the PR to classify to level 2 (each extension to their entity):

![image](https://user-images.githubusercontent.com/683500/234705144-ef061f0e-a492-4e59-a5ba-73896bf50b2a.png)